packages/e2e-tests/specs/editor/blocks/__snapshots__/heading.test.js.snap

```diff
@@ -2,25 +2,25 @@
 
 exports[`Heading can be created by prefixing existing content with number signs and a space 1`] = `
 "<!-- wp:heading {\\"level\\":4} -->
-<h4>4</h4>
+<h4 id=\\"wp-4\\">4</h4>
```

Comment:
Why prefix with `wp-`?

Reply:
@ellatrix: that's, as of now, the prefix to mark auto-generated anchors. See https://github.com/WordPress/gutenberg/pull/30825#discussion_r625788775

Reply:
Sure, but why prefix at all? What value does it add?

Reply:
The only reason for prefixes was that I couldn't find a way to make it work properly without them. Resolved in https://github.com/WordPress/gutenberg/pull/30825/commits/f1ad7c2cc61795fb632f53c9cdbbd6686d62e7d9, prefixes were removed :+1: